builds/misc/mqtt-perf.yaml

```diff
@@ -0,0 +1,78 @@
+# MQTT Performance tests pipeline
+# Each platform separated into stages
+# Each stage consists of 
+# - setup job for test host (pull and start 'mqttd' container). See templates/mqtt-perf-setup.yaml
+# - setup job for publisher node and subscriber node (mqtt-benchmark tool writen in go). See templates/mqtt-perf-setup.yaml
+# - a set of test cases. See templates/mqtt-perf-test-case.yaml.
+
+# Pipeline requires custom agent pool (defined in $(pool.name)) 
+# with at least two separate agents:
+# - with 'mqtt-perf-tests-pub' demand - for publisher node
+# - with 'mqtt-perf-tests-sub' demand - for subscriber node
+# And two SSH service connections for AMD64 and ARM32 test hosts:
+#  - $(broker.sshconnection.amd64)
+#  - $(broker.sshconnection.arm32v7)
+# to manage and run 'mqttd' docker container there.
+
+trigger: none
+pr: none
+
+# github repo for our fork of mqtt-benchmark tool.
+resources:
+  repositories:
+  - repository: mqttbenchmark
+    type: github
+    endpoint: Azure/iotedge
+    name: vadim-kovalyov/mqtt-benchmark
```

Comment:
is this your branch?

Reply:
This is a fork of mqtt-benchmark tool. I discussed with the team if it make sense to put it into our repo, and we decided against it, since this test tool is not really related to iotedge platform.

Reply:
I think we should put into our repo.
1. First IoT Edge is open-source.  It is very welcome to other devs outside MS to contribute.  Therefore we should allow a way for people to run any kind of tests we had, e.g. CI, E2E, LH, stress and connectivity, therefore perf test should be.
2. I believe we are going to share the benchmark of MQTT broker feature.  Then how we are going to do benchmark should be transparent to customers and we should document down and share in github repo.
3. We do have some code is not directly related to IoT Edge product in our github repo.  For example, test dashboard, snitcher and all other test modules.

Reply:
I don't mind either way, I'll let @varunpuranik and @dmolokanov comment on this as well.

Reply:
@philipktlin These are good questions. But even without mqtt-bench tool we have `cross` fork that live in @arsing account, some patched rust crates that live in @philipktlin account etc... I don't mind move it under one repo, but 
1. we need to do that across all dependencies
2. probably we need to make additional repo `iotedge-infra` and put all 3rd party tools there

Reply:
Those are different reasons to leave there and we really want to get rid of them.  Please create a PBI to track this suggestion and ask Varun to make the final call here.  thanks.  :)

Reply:
I have created a PBI to track this question.